habitat-baselines/habitat_baselines/config/habitat_baselines/rl/policy/monolithic_pointnav.yaml

```diff
@@ -0,0 +1,4 @@
+name: "PointNavResNetPolicy"
```

Comment:
let's call this just "monolithic.yaml". The PointNavResNet is just a ResNet policy (initially used for PointNav, but used for other stuff as well now) that we should rename in a future PR.